lib/Minify/App.php

```diff
@@ -118,13 +129,13 @@ public function __construct($dir)
         };
 
         $this->env = function (App $app) {
-            return new \Minify_Env($app->config->envArgs);
+            return new Minify_Env($app->config->envArgs);
         };
 
         $this->errorLogHandler = function (App $app) {
             $format = "%channel%.%level_name%: %message% %context% %extra%";
-            $handler = new \Monolog\Handler\ErrorLogHandler();
-            $handler->setFormatter(new \Monolog\Formatter\LineFormatter($format));
+            $handler = new Monolog\Handler\ErrorLogHandler();
```

Comment:
Didn't want to add these uses? Fine with me either way.

Reply:
yeah. it became too ambiguous which one is imported